flaskr/app.py

```diff
@@ -7,6 +7,7 @@
 from graphene_sqlalchemy import SQLAlchemyObjectType, SQLAlchemyConnectionField
 from flask_graphql import GraphQLView
 from flask_cors import CORS
+from sqlalchemy import desc
```

Comment:
Wondering about the full meaning of this.  Is this just for the query?

Reply:
Yup. Desc needs to be specifically imported from sqlalchemy, you can't call it if you don't import it. Kind of like how you have to import certain math functions from modules to use them.